flink-queryable-state/flink-queryable-state-runtime/src/test/java/org/apache/flink/queryablestate/itcases/NonHAQueryableStateFsBackendITCase.java

```diff
@@ -86,10 +86,10 @@ private static Configuration getConfig() {
 		config.setInteger(QueryableStateOptions.SERVER_NETWORK_THREADS, 1);
 		config.setString(
 			QueryableStateOptions.PROXY_PORT_RANGE,
-			QS_PROXY_PORT_RANGE_START + "-" + (QS_PROXY_PORT_RANGE_START + NUM_TMS));
+			QS_PROXY_PORT_RANGE_START + "-" + (QS_PROXY_PORT_RANGE_START + 100));
 		config.setString(
 			QueryableStateOptions.SERVER_PORT_RANGE,
-			QS_SERVER_PORT_RANGE_START + "-" + (QS_SERVER_PORT_RANGE_START + NUM_TMS));
+			QS_SERVER_PORT_RANGE_START + "-" + (QS_SERVER_PORT_RANGE_START + 100));
```

Comment:
I think 100 is enough, just a suggestion: add a static variable， such as: `private static final int NUM_PORT_COUNT = 128;` What do you think?

Reply:
Make sense!